src/kontrol/foundry.py

```diff
@@ -746,12 +748,18 @@ def foundry_summary(
 
     main_file = output_dir / Path(name + '.sol')
 
+    if license is None:
+        license = 'UNLICENSED'
+    else:
+        if not license:
+            raise ValueError('License cannot be empty')
```

Comment:
With `license` defaulting to `UNLICENSED`, then you only have to check for the empty string, or a string of whitespaces.
```suggestion
    if not license.strip():
        raise ValueError('License cannot be empty or blank')
```